app/controllers/miq_request_controller.rb

```diff
@@ -95,6 +95,8 @@ def show
     @display = params[:display] || "main" unless pagination_or_gtl_request?
     @gtl_url = "/show"
 
+    @request_tab = params[:typ] || 'service'
+
```

Comment:
Everything looks great! One suggestion is to delete this extra empty line.

Reply:
I was thinking about it but there is an if statement underneath that and I just wanted to separate them a little bit.